Jenkinsfile

```diff
@@ -108,9 +108,15 @@ node (node_label) {
 			load "pipeline-scripts/workload.groovy"
 		}
 		if (ocpv4_scale == "TRUE") {
-			env.WORKLOAD_PROPERTIES_FILE=OPENSHIFTv4_SCALE_PROPERTY_FILE
-			env.WORKLOAD="ATS-SCALE-CI-SCALE"
-			load "pipeline-scripts/workload.groovy"
+			if (openshiftv4_install_on_bm == "TRUE") {
+				env.WORKLOAD_PROPERTIES_FILE=OPENSHIFTv4_ON_BM_PROPERTY_FILE
```

Comment:
Maybe rename OPENSHIFTv4_ON_BM_SCALE_PROPERTY_FILE

Reply:
I am using the same file for both deploy and scaleup jobs, as they are 99% same just one extra environment variable `OPENSHIFT_WORKER_SCALEUP_TARGET`. 
Do you want to use a different property file for scaleup ?

Reply:
ha that's fine then

Reply:
Actually thinking again....I still think we need a new property file because we won't change the property file once we kick off the pipeline right. The pipeline will have both the deploy job as well as the scale up job. We need a property file that has the worker count for the scale up defined.

Reply:
Still think it's better to have it's own property file to avoid any confusion.

Reply:
Also add an example to properties-files directory.

Reply:
Ignore my comment, we don't really need a new properties-file.